CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [2.1.1] - unreleased
 
+- pueue log output now includes the task label, if any. [#355](https://github.com/Nukesor/pueue/issues/355)
+
```

Comment:
Could you move this to the `### Added` section

Reply:
There's so much stuff in the current release, you can change the next release from `2.1.1` to `2.2.0` as well while you're at it :D 

This somehow mutated from a point release to a proper minor feature release.